src/main/java/ch/njol/skript/expressions/ExprLastSpawnedEntity.java

```diff
@@ -99,8 +125,9 @@ public boolean isSingle() {
 	}
 	
 	@Override
-	public String toString(final @Nullable Event e, final boolean debug) {
-		return "the last " + (from == 1 ? "spawned" : from == 1 ? "shot" : "dropped") + " " + type;
+	public String toString(@Nullable Event e, boolean debug) {
```

Comment:
maybe it'd be nicer to do a switch statement here? would be nice if anything else is added to this expression in the future too.